docs/developer/architecture.rst

```diff
@@ -246,6 +242,68 @@ documentation page on :doc:`contributing </contributing>`.
     - Uses the Ray_ execution framework.
     - The compute kernel/in-memory format is a pyarrow Table.
 
+Module/Class View
+-----------------
+Modin modules layout is shown below. To deep dive into Modin internal implementation 
+details just pick module you are interested in (only some of the modules are covered 
+by documentation for now, the rest is coming soon...).
+
+.. parsed-literal::
+   ├───.github
+   ├───asv_bench
+   ├───ci
+   ├───docker
+   ├───docs
+   ├───examples
+   ├───modin
+   │   ├───backends
+   │   │   ├───base
+   │   │   │   └───query_compiler
+   │   │   ├───pandas
+   │   │   |   ├─── :doc:`parsers </flow/modin/backends/pandas/parsers>`
+   │   │   │   └───query_compiler
+   │   │   └───pyarrow
+   │   │   |   ├───parsers
+   │   │   │   └───query_compiler
+   │   ├───config
+   │   ├───data_management
+   │   │   ├───factories
+   │   │   └───functions
+   │   ├───distributed
+   │   │   └───dataframe
+   │   │       └───pandas
+   │   ├───engines
+   │   │   ├───base
+   │   │   │   ├───frame
+   │   │   │   ├─── :doc:`io </flow/modin/engines/base/io>`
+   │   │   │   └─── :doc:`series </flow/modin/engines/base/series>`
```

Comment:
Since we removed stuff related to internal Series object this should be removed as well.

Reply:
`series` mentioning was removed.